DataCollector/src/main/java/com/braintreepayments/api/datacollector/DataCollector.kt

```diff
@@ -100,10 +99,11 @@ class DataCollector @VisibleForTesting internal constructor(
             if (configuration != null) {
                 val deviceData = JSONObject()
                 try {
-                    val internalRequest = DataCollectorInternalRequest(request.hasUserLocationConsent)
-                        .setApplicationGuid(getPayPalInstallationGUID(context))
+                    val internalRequest = DataCollectorInternalRequest(request.hasUserLocationConsent).apply {
+                        applicationGuid = getPayPalInstallationGUID(context)
+                    }
                     if (request.riskCorrelationId != null) {
-                        internalRequest.setRiskCorrelationId(request.riskCorrelationId)
+                        internalRequest.clientMetadataId = request.riskCorrelationId
```

Comment:
qq: Was the goal with renaming this property from `riskCorrelationId` to `clientMetadataId` to make things clearer?

Reply:
In the original implementation of DataCollectorInternalRequest, the setter's parameter has a different name from the internal parameter's, however, the getter returns the internal parameter's name. Therefore, I used the internal parameter's name as the Kotlin property's name and kept the function caller's parameters the same.

public DataCollectorInternalRequest setRiskCorrelationId(@NonNull String riskCorrelationId) {
        this.clientMetadataId = riskCorrelationId.substring(0, Math.min(riskCorrelationId.length(), 32));

        return this;
    }
    
    String getClientMetadataId() {
        return clientMetadataId;
    }

Reply:
Ok yeah that makes sense. There's a lot of discrepancy with those synonymous terms. I agree with `clientMetadataId` though it seems to work well here.

Reply:
Can we call out this change in the changelog? this would be a breaking change for merchants.